code/game/turfs/simulated/walls_misc.dm

```diff
@@ -64,10 +64,12 @@
 /turf/closed/wall/clockwork/process()
```

Comment:
If it was really that significant then you'd probably notice once ratvar spawns but you don't so clearly it can't be that bad, right?